doc/getting_started.md

```diff
@@ -630,7 +610,7 @@ Run the following commands
 
 ```console
 $ cd cw/cw305
-$ ./simple_capture_traces_batch.py --scope cw
+$ ./capture.py --cfg-file capture_aes.yaml capture aes-random-batch
```

Comment:
Here we would need an argument `--num-traces 2000000`

Reply:
Actually, you now wrote `--num-traces=2000000` does that work too or do we need `--num-traces 2000000` (without equal sign)?

Reply:
My mistake :) Changed it.